### DISCUSSION TITLE
[Feature] Save edited photos to folder that is synced with immich

### DISCUSSION THREAD

**Discussion by tschoesi** [immich-app/immich#13635] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

> Following might be android specific, don't know how it works on iOS.

When images are saved after editing, they go to a directory called `Pictures`. This is a folder that is used by various other apps as well.

I suggest using a separate, immich specific directory. 
This would allow a user to back up this directory with immich without also ending up with the images from other apps.

This immich specific directory could also be backed up by default. Currently, edited images don't appear in immich, which seems weird to me.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6439
